templates/template_index.html

```diff
@@ -184,10 +184,26 @@ <h4><i class="fas fa-expand-alt feature-icon"></i> Render Scaling</h4>
 
 <hr>
 <div class="embed-responsive embed-responsive-16by9">
-  <iframe class="embed-responsive-item" src="https://www.youtube-nocookie.com/embed/?list=PLec5KNGR_nILyzdbQ8OyFpceyrTRUBNNU&autoplay=1&mute=1&modestbranding=1&loop=1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  <iframe class="embed-responsive-item" src="https://www.youtube-nocookie.com/embed/?list=PLec5KNGR_nILyzdbQ8OyFpceyrTRUBNNU&autoplay=0&mute=1&modestbranding=1&loop=1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 <hr>
 
+<div class="" id="screenshots">
```

Comment:
* Let's move screenshots into a subdirectory
* Empty `class`
* It would be nice to have the [screenshots in a carousel](https://getbootstrap.com/docs/5.3/examples/carousel/), but we can do it later

Reply:
I was not sure about the empty class, copied from `<div class="" id="compatibility">`

Reply:
Both are now removed, screenshots moved to `resources/screenshots`.